.github/workflows/build-macos.yml

```
@@ -1,4 +1,4 @@
-name: Build MacOS 11
+name: Build for MacOS
```

```suggestion
name: Build for macOS
```

It's lowercase macOS nowadays. :)